activerecord/lib/active_record/connection_adapters/abstract_adapter.rb

```diff
@@ -174,6 +175,14 @@ def schema_migration # :nodoc:
                               end
       end
 
+      def prepared_statements # :nodoc:
+        @prepared_statement_status.value
+      end
+
+      def prepared_statements=(prepared_statements) # :nodoc:
```

Comment:
I think this can (should) be private -- no external callers

Reply:
Good point. Changed.